code/src/components/Movie.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable max-len */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/jsx-closing-bracket-location */
+/* eslint-disable quotes */
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Movie = (props) => {
+  return (
+    <div className="movie-container">
+      {/* ett snyggt grid på movie-container, tänk på att den ska kunna vara responsiv vid mindre än desktop */}
+      <Link key={1} to="toBeDeclared" className="movie">
```

Comment:
Key should be edited, I totaly missed that before committing. I think the best key is the id value in the array